tool/tctl/common/loadtest_command.go

```diff
@@ -76,6 +81,8 @@ func (c *LoadtestCommand) Initialize(app *kingpin.Application, config *servicecf
 
 	c.watch = loadtest.Command("watch", "Monitor event stream").Hidden()
 	c.watch.Flag("kind", "Resource kind(s) to watch, e.g. --kind=node,user,role").StringVar(&c.kind)
+	c.watch.Flag("ops", "Operations to watch, e.g. --ops=put,del").Default("put,del").StringVar(&c.ops)
+	c.watch.Flag("format", "Output format").Default(teleport.Text).StringVar(&c.format)
```

Comment:
Should this be an `EnumVar`?

Same goes for `ops`.

Reply:
Updated `format` to be an `EnumVar`.  `ops` doesn't map as nicely though since it's intended to be a comma-separated list.

Reply:
Isn't that what `EnumsVar` is for?

Reply:
`EnumsVar` allows for something like `tctl loadtest watch --ops=del --ops=put` but doesn't support lists like `tctl loadtest watch --ops=put,del`.  The use of comma-separated list parameters is the goto way to handle flags that can take multiple values across the rest of teleport (token types, request ids, label selectors, etc).  I'd rather keep it consistent.

Reply:
Got it. I had hoped it would support comma separated, as I agree that's more ergonomic.